src/include/daos_srv/bio.h

```diff
@@ -43,7 +43,8 @@ typedef struct {
 	uint16_t	ba_type;
 	/* Is the address a hole ? */
 	uint16_t	ba_hole;
-	uint32_t	ba_padding;
+	uint16_t	ba_dedup;
+	uint16_t	ba_padding;
```

Comment:
we should update POOL_DF_VERSION due to format change

Reply:
updated POOL_DF_VERSION